groups/templates/groups/group_edit.html

```diff
@@ -1,4 +1,7 @@
 {% include 'base.html' %}
+{% block content %}
+{% load tz %}
+{% timezone "US/Pacific" %}
```

Comment:
This is a good start. Could the `timezone` value be set in a config file somewhere.